docs/configuration.md

```diff
@@ -213,6 +213,15 @@ of the most common options to set are:
     and typically can have up to 50 characters.
   </td>
 </tr>
+<tr>
+  <td><code>spark.driver.supervise</code></td>
+  <td>false</td>
+  <td>
+    If value set true, restarts the driver on failure, make sure that
```

Comment:
This sentence needs to be simplified. "If true, restarts the driver automatically if it fails with a non-zero exit status."